controllers/drplacementcontrol_controller.go

```diff
@@ -59,6 +59,9 @@ const (
 	MaxPlacementDecisionConflictCount = 5
 
 	DestinationClusterAnnotationKey = "drplacementcontrol.ramendr.openshift.io/destination-cluster"
+
+	DoNotDeletePVCAnnotation    = "drplacementcontrol.ramendr.openshift.io/do-not-delete-pvc"
+	DoNotDeletePVCAnnotationVal = "true"
```

Comment:
We already have another constant for "true" - should we have one constant for this value instead of adding new "true" constant for every new annotation?

Reply:
Probably, but that value meant to hold something else.  It is not necessary to have it to true.  That was just a first thing that came to my mind.  The code handles any string, that's why for now, it is simply set to the string "true".

Reply:
"true" seems popular and good value for boolean-like annotation without meaningful value. Since we already have such annotations, using single constant for all boolean-like annotation would be nice.

Reply:
I disagree. The annotation is a map entry of string/string. You are looking at it as a boolean value which can be reused. I am looking at it as a value that can give more meaningful description later on.  Even now I can name it something like `"volsync managed pvcs only"`. It is a pair of string/string and I believe they should go together.